style.css

```diff
@@ -0,0 +1,178 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Lexend+Deca&display=swap');
+
+:root {
+  --grey: #979797;
+  --dark: #1B1937;
+  --white: #FFFFFF;
+  --purple: #AB5CDB;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+}
+body {
+  background-color: black;
+}
+
+main, footer {
+  width: 327px;
+  display: flex;
+  margin: 0 auto;
+  margin-top: 30px;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+}
+
+h1, dd {
```

Comment:
You will get more maintainable, easy to read & modify CSS if you define rules for one element at a time.
Another good practice is defining global styles first (e.g. for `h1`, `p`, `a`, `img`), then defining the more specific classes (`footer p`, `dl.stats-items`, `.stats-item dt`, `.stats-item dd` etc).